client/my-sites/site-settings/search.jsx

```diff
@@ -69,22 +70,55 @@ class Search extends Component {
 		);
 	}
 
-	renderSettingsContent() {
-		const { activatingSearchModule, searchModuleActive, translate } = this.props;
+	renderSettingsContent( activatingSearch, searchActive ) {
+		const { translate } = this.props;
 
 		return (
 			<div className="search__module-settings site-settings__child-settings">
-				{ activatingSearchModule && (
+				{ activatingSearch && (
 					<FormSettingExplanation>{ translate( 'Activating search…' ) }</FormSettingExplanation>
```

Comment:
This "Activating search..." shows even when deactivating search for WordPress.com sites, which differs from how it works for Jetpack sites. On Jetpack sites, it only shows "Activating search..." when actually activating search. When deactivating search for a Jetpack site, the explanation is "Add the Search widget to your sidebar to configure advanced search filters." until the module is disabled, after which there is not an explanation.